.gitignore

```diff
@@ -39,3 +39,6 @@ ENV/
 # logs
 *.log
 .pytest_cache/
+
+# vim
+*.swp
```

Comment:
It's not project specific and imo that's why it should be kept in developer own global git ignore file.